about.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: page
+title: Discovery Environment Docs
+---
+
+The [Discovery Environment (DE)](http://www.cyverse.org/ci/discovery-environment) is the primary 
+web interface and platform to access the powerful computing, storage, and analysis application 
+resources of the [CyVerse](http://www.cyverse.org) cyberinfrastructure.
+
+# Useful Links Links
+
+* [CyVerse Main Page](http://www.cyverse.org/)
+* [Discovery Environment Main Page](http://www.cyverse.org/ci/discovery-environment)
+* [DE Docker Hub](https://hub.docker.com/r/discoenv/)
+* [DE User Manual](https://pods.iplantcollaborative.org/wiki/display/DEmanual/Table+of+Contents)
+* [Ask iPlant](http://ask.iplantcollaborative.org/questions/)
+* [iPlant Wiki](https://pods.iplantcollaborative.org/wiki/dashboard.action)
+* [iPlant Facebook](https://www.facebook.com/iPlantCollab/)
+* [iPlant Twitter](https://twitter.com/iPlantCollab)
```

Comment:
Good point, but I left them that way on purpose. Kinda hesitant to make the switch until the backing service doesn't say "iPlant".

Still, when I'm done with this, I'll have to make a final pass for consistency, one way or the other.